thirdparty/patches/glog-suffix-log.patch

```diff
@@ -0,0 +1,13 @@
+diff --git src/logging.cc src/logging.cc
+--- src/logging.cc
++++ src/logging.cc
+@@ -1135,7 +1135,8 @@ void LogFileObject::Write(bool force_flush,
+                     << setw(2) << tm_time.tm_min
+                     << setw(2) << tm_time.tm_sec
+                     << '.'
+-                    << GetMainThreadPid();
++                    << GetMainThreadPid()
++                    << ".log";
```

Comment:
I'm not sure what this fixes. Can you add a comment here?

Reply:
I just make sure this change will be same log file suffix with previous.

Reply:
I see that you've deleted `StreamLogger`. Do we still need this patch? What's the pattern of log file names if this patch is not applied?

Reply:
I think I only used `StreamLogger` to put all the log severities into the same file? I think it also used to be used for something else, but otherwise there was nothing special about it.